docs/serialization.md

```diff
@@ -175,6 +175,7 @@ The final argument to `pm_serialize_parse` is an optional string that controls t
 | ...     | the encoding bytes         |
 | `1`     | frozen string literal      |
 | `1`     | suppress warnings          |
+| `1`     | syntax version, see [pm_options_version_t](https://github.com/ruby/prism/blob/main/include/prism/options.h) for valid values |
```

Comment:
"syntax version" is  my desired interpretation of it, feel free to suggest to something else.
Just "version" or "prism version" seems too confusing as it's not the same thing as the gem version.